Gemfile

```diff
@@ -10,4 +10,6 @@ gem 'listen'
 gem 'bundler-audit'
 gem 'rubocop'
 gem 'rails-decorators', git: 'https://github.com/workarea-commerce/rails-decorators.git'
-gem 'teaspoon', git: 'https://github.com/jtapia/teaspoon.git', branch: 'chore/update-rails-6'
+
+gem 'teaspoon', git: 'https://github.com/jejacks0n/teaspoon.git'
+gem 'coffee-script', group: :test
```

Comment:
From that issue, looks like it was teaspoon fixed years ago, things run fine on my local copy without this. What was the weirdness?

Reply:
My local was hanging when running the real test suite. I found the dependency after running `teaspoon:server`. It was printing out a unmet dependency error around coffee-script. This cleared it up for me, but I'll re-check without it.

Reply:
Dunno what was going on in my brain